### DISCUSSION TITLE
where can i find dex.yaml file?

### DISCUSSION THREAD

**Discussion by divinesocial** [dexidp/dex#2496] [Q&A]
below documentation contains script to create a kubernetes component `kubectl create -f dex.yaml`
But i'm not able to find content for dex.yaml file. Please let me know where can i find the file.

https://dexidp.io/docs/kubernetes/#deploy-the-dex-server
![image](https://user-images.githubusercontent.com/104025959/165508274-4919f199-f508-428d-909b-ea0935c70287.png)

**Comment by nabokihms** [comment]
You can find it in the example folder https://github.com/dexidp/dex/tree/master/examples/k8s

P.S. It is also possible to install dex on top of Kubernetes using our official helm chart https://artifacthub.io/packages/helm/dex/dex
